### ISSUE TITLE
Unable to redo the oAuth process

### ISSUE THREAD

**Issue by m-p-3** [jamesgrayillustration/tinyreel#1]
Hi,
I've been having some problem to get the application to work after authenticating to Instagram, so I decided the revoke access to the application and go through the setup again.

Well, it looks like the settings webpage doesn't see that the app doesn't have access to my Instagram account anymore, and shows my avatar and the button "Ok, close this!".

I tried removed and readding the watchapp, but it doesn't solve the issue. I guess I'm stuck somehow.

**Comment by grayxr** [comment]
Hi @m-p-3 - thanks for bringing this to my attention - the auth info is cached for 24 hrs. I guess I can make a logout feature and/or make a 'remember me' option. The web-app handles the oAuth login transaction, so that's why the methods you used didn't affect it. I've been having several users indicate that they get stuck on the 'Loading' screen, and i'm not sure what the culprit is, but I suspect it has something to do with fetching the user feed that's breaking it (timeouts, empty values, maybe strange characters, etc.). I have a separate repo for the web-app.

**Comment by m-p-3** [comment]
I managed to reauthenticate, so I guess that is somewhat sorted out. Now, the watchapp seems to be stuck at loading anything, but I don't want to open a bug report without anything tangible to work with. I'll be digging in the log files and see if I can find anything relevant.

**Comment by grayxr** [comment]
@m-p-3, yeah if you can manage to find the ("Loading..." indefinitely) bug, let me know - as I am extremely busy at the moment and don't have a lot of time to dedicate to this. If you want me to help to get you setup to use the web-app locally, let me know.
